routers/weatherRouters.go

```diff
@@ -0,0 +1,7 @@
+package routers

```

Comment:
@carojaspy what's the purpose of this file?

Reply:
It was a intent to split the routers as the beego API starter project, but you help me the last week, it was necessary to create the router with **bee generate**, I'll deleted this file from this branch.